common_functions.sh

```diff
@@ -235,6 +240,11 @@ function build_tags() {
 	do
 		for arch in ${arches}
 		do
+			# Check if all the supported arches are available for this build.
+			sup=$(vm_supported_onarch ${vm} ${shasums} ${arch})
```

Comment:
what is a sup :-)

Reply:
sup as in ... sup dude ? :)
Fixed now !